entrypoint

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This command takes one positional argument, `command`, which specifies whether
+to initialize test repos locally, or to start an hg serve comand serving the
+test repos.
+"""
+
+import argparse
+import subprocess
+import os
+from pathlib import Path
+
+
+BASE_REPO_DIR = Path("/repos")
+
+HG_SERV_PORT = os.environ.get("PORT", "8000")
```

Comment:
typo: `$HG_SERVE_PORT`

Reply:
lol -- I actually had intentionally used "SERV" here as an abbreviation, but I agree that HG_SERVE_PORT is better!